docs/en/contributing.md

```diff
@@ -0,0 +1,61 @@
+# Contributing
```

Comment:
That may go into https://github.com/yiisoft/db/blob/master/.github/CONTRIBUTING.md